react/Geolocation.js

```diff
@@ -31,16 +35,33 @@ class Geolocation extends Component {
   }
 
   componentDidMount() {
+    this.startTime = Date.now()
```

Comment:
can you expand a bit more on why this start time is being set only on did mount? I'm still trying to understand the components and changes 😅

Reply:
@lucasecdb Good question, the intention here was to measure how long the whole process takes, including the time the user takes to decide whether to click on "Allow" or "Deny" on the geolocation prompt, which is why it's starting on `componentDidMount` instead of down below. 

Not 100% sure about that though, but perhaps we could measure both

Reply:
oh, I see! thanks for clarifying